theme_crowdfunding/views/assets.xml

```diff
@@ -2,6 +2,7 @@
 <odoo>
     <template id="theme_crowdfunding_assets" name="Assets for crowdfunding website" inherit_id="website.assets_frontend">
         <xpath expr="link[last()]" position="after">
+            <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous"/>
```

Comment:
Why do you need fontawesome css, isn't it included with Odoo?

Reply:
I didn't know they were, removed it.